lib/centurion/mock.rb

```diff
@@ -0,0 +1,49 @@
+module Centurion
+  class Mock
+    def initialize(opts)
+      environment = opts[:current_environment]
+      @options = opts[environment]
+    end
+
+    def run
+      require 'pry-byebug'
```

Comment:
Absolutely not! I made the PR because I was using it and wanted to share what I was doing to gauge if there was an interest. If there was, I wanted to actually continue and write some tests to it as well. Maybe name it differently that `Mock`. I actually did not think it was going to go through all the way at this stage. 

That was what I was trying to say in the PR message initially when saying: 
> This is a work in progress, and was wondering if it was worth it or not to have me continuing, testing, etc...